.github/workflows/sanitizers.yml

```diff
@@ -48,9 +40,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: recursive
+      - name: Create Couchbase cluster with cbdinocluster
+        id: cbdino
+        uses: ./.github/actions/create-cluster
+        with:
+          version: ${{ matrix.server }}
       - name: Initialize couchbase
         env:
           CB_TRAVEL_SAMPLE: yes
+          CB_HOST: ${{ steps.cbdino.outputs.ip }}
         run: ./bin/init-cluster
```

Comment:
Maybe we do not really need `bin/init-cluster` anymore?

Ruby SDK works without it just fine:
https://github.com/couchbase/couchbase-ruby-client/blob/8574a1a1d2c7a2687188af5dea7d29c522c4c37e/.github/workflows/tests.yml#L648-L672

It can create buckets, collections, etc.
```
          cbdinocluster -v buckets add ${CLUSTER_ID} default --ram-quota-mb=100 --flush-enabled=true
          cbdinocluster -v buckets load-sample ${CLUSTER_ID} travel-sample
```